apps/src/p5lab/poetry/poem.js

```diff
@@ -7,8 +7,9 @@ export function getPoem(key) {
   }
   return {
     key: key,
+    locale: POEMS[key].locale,
     author: POEMS[key].author,
-    title: msg[`${key}Title`](),
-    lines: msg[`${key}Lines`]().split('\n')
+    title: POEMS[key].title || msg[`${key}Title`](),
+    lines: POEMS[key].linesSplit || msg[`${key}Lines`]().split('\n')
```

Comment:
will this break translated poems? i would think this logic would mean that whatever is in poetry/constants.js is always used, but maybe i'm misreading it

Reply:
oh, nevermind -- it looks like `title` and `linesSplit` will only exist for poems that shouldn't be translated, so all translated poems should stay as-is

Reply:
sorry for all the comments, but another thought 😅 we should add this info to the comment on line 29 (e.g., if the poem is translated, just author is needed here; otherwise...)

Reply:
Good point, I'll update the comment to describe these details.